src/addonloader/luaaddonstate.h

```diff
@@ -123,6 +123,14 @@ class LuaAddonState {
     // @function currentInputMethod
     // @treturn string the unique string of current input method.
     DEFINE_LUA_FUNCTION(currentInputMethod);
+    //
```

Comment:
Please add document for these two function

Reply:
Thanks, I will add it